packages/perseus/src/widgets/input-number/input-number.test.ts

```diff
@@ -237,7 +237,7 @@ describe("input-number", function () {
         if (!transform) {
             throw new Error("transform not defined");
         }
-        const widgetProps = transform(editorProps, mockStrings);
```

Comment:
This function doesn't take a `strings` object at all.

Reply:
When you go to the `WidgetTransform` type (by command clicking `transform` here in GoLand), it does have strings as a parameter.
![image](https://github.com/user-attachments/assets/090884dd-0cb3-43c6-8cfe-54e6fe92ffd3)

But when you go to the transform for inputNumber, it just takes the input number widget options.
![image](https://github.com/user-attachments/assets/72282087-19f5-40b2-8a79-aef97b02db6d)

Should strings on the `WidgetTransform` type be optional? Why doesn't this cause an error?

Reply:
You are correct that the `WidgetTransform` function type has a `strings` parameter. However, in this test we've picked the `transform` function from the `default` export of the `input-number.tsx` file. That function is compatible with the `WidgetTransform` function type, but it ignores the `strings` param. In Javascript/TypeScript, functions can chose to drop any arguments from the right side of the declaration and still be considered compatible. 

So this is valid:

```ts
type WidgetTransform = (options: WidgetOptions, strings: PerseusStrings, problemNum?: number) => any;

const transform: WidgetTransform = (options) => { return {...options} };
```